Ratings_MVP/Ratings_MVPUITests/RatingsMVPUITests.swift

```diff
@@ -0,0 +1,39 @@
+// RatingsMVPUITests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import XCTest
+
+/// UI tests
+final class RatingsMVPUITests: XCTestCase {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let tableId = "movieListTable"
+        static let numberZero = "0"
+        static let eightDotSeven = "8.7"
+        static let topRaiting = "Топ рейтинга"
+        static let premiers = "Премьеры"
+        static let populars = "Популярные"
+        static let viewName = "Ratings_MVP.MovieDetailsView"
+        static let back = "Back"
+        static let catInBoots = "Кот в сапогах 2: Последнее желание"
+        static let movieListTable = "movieListTable"
+        static let movieDetailsView = "Ratings_MVP.MovieDetailsView"
+    }
+
+    // MARK: - Public Methods
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+    }
+
+    func testExample() throws {
+        let app = XCUIApplication()
```

Comment:
```suggestion
        let app = XCUIApplication()
```
а если тебе еще один тест понадобиться, опять будешь создавать app? Я бы вынесла его наверх